src/App.css

```diff
@@ -1,6 +1,6 @@
-.App {
-  position: relative;
-  width: 100%;
-  height: 100%;
+html,
+body {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen', 'Ubuntu', 'Cantarell', 'Fira Sans',
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected newline after "," in a multi-line list (value-list-comma-newline-after)](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5592511)